tests/features/bootstrap/FeatureContext.php

```diff
@@ -212,4 +212,42 @@ public function createLanguages($langcode) {
     $this->languageCreate((object) ['langcode' => $langcode]);
   }
 
+  /**
+   * Prepare for PHP errors log.
+   *
+   * @BeforeSuite
+   */
+  public static function preparePhpErrors($event) {
+    // Clear out the watchdog table at the beginning of each test suite.
+    db_truncate('watchdog')->execute();
+  }
+
+  /**
+   * Check for PHP errors log.
+   *
+   * @AfterSuite
+   */
+  public static function checkPhpErrors($event) {
+    // Find any PHP errors at the end of the suite
+    // and output them as an exception.
+    $log = db_select('watchdog', 'w')
+      ->fields('w')
+      ->condition('w.type', 'php', '=')
+      ->execute()
+      ->fetchAll();
```

Comment:
Strange that the db_log module doesn't give us an API to get the data. The only solution seems to indeed query the database directly.